src/routes/(app)/tag/[id]/+page.svelte

```diff
@@ -9,6 +9,13 @@
 		class="w-full rounded-xl body-secondPlan border-gray-300 border py-3 px-3 flex flex-col gap-2"
 	>
 		<TagWidget tag={data.tag} displayOption="var-1" pictures={data.pictures} />
+		<TagWidget
+			tag={data.tag}
+			displayOption="var-1"
+			titleClassName="regular"
```

Comment:
uhm no 

- CMS should pass `titleCase` to `TagWidget`
- `TagWidget` should pass `titleClassName` to its variations

check again my comments in the previous review